Readme.md

```diff
@@ -1,9 +1,33 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128596817/21.2.1%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T566419)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# How to use the Web Document Viewer in JavaScript with Angular
+# Reporting for Angular - How to integrate the Web Document Viewer in Angular App
+
+This example shows how to incorporate Web Document Viewer into a client-side app built with Angular. The example consists of two parts:
+
+- A server (back-end) ASP.NET Core project that enables [cross-domain requests (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) (Access-Control-Allow-Origin) and implements a custom web report storage.
+
+- An [Angular](https://angular.io/) front-end client application.
+
+## Run the Example
+
+Perform the following steps to run this example:
+
+1. Open the back-end project solution (**ServerSideApp.sln**) in Visual Studio and run the project.
+2. Navigate to the **angular-document-viewer** folder that is the client part's root folder.
+3. Open the console and run the following command to download packages:
+
+    ```npm install```
+
+4. Run the command to compile and start the client part:
+
+    ```npm start```
+
+5. Point your browser to [http://localhost:4200/](http://localhost:4200/) to see the result.
+
+![](images/screenshot.png)
+
 
 ## Upgrade the Project
 

```

Comment:
On line 10 below, please change "Front-end and Back-end" to "front-end and back-end".